packages/dev-ui/src/tables/serializeWithoutIndexedValues.ts

```diff
@@ -0,0 +1,13 @@
+export function serializeWithoutIndexedValues(obj: any) {
+  return JSON.stringify(obj, (key, value) => {
+    // strip indexed values
+    if (/^\d+$/.test(key)) {
+      return;
+    }
+    // serialize bigints as strings
+    if (typeof value === "bigint") {
+      return value.toString();
+    }
+    return value;
+  });
```

Comment:
would it be worth to use the other `serialize` util here?

Reply:
this is temporary until we're rendering table with columns instead of just dumping json